sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_secret.hpp

```diff
@@ -10,6 +10,11 @@
 #include "azure/keyvault/secrets/keyvault_secret_properties.hpp"
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
+
+  /**
+   * @brief A Secret held in KeyVault.
+   *
+   */
   struct KeyVaultSecret
   {
```

Comment:
I believe this fixes https://github.com/Azure/azure-sdk-for-cpp/issues/4691

Consider borrowing the wording from Java, which reads pretty well, imo.

Here's the doc description from the rest-api-spec:
https://github.com/Azure/azure-rest-api-specs/blob/c7f3e601fd326ca910c3d2939b516e15581e7e41/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.4/keys.json#L1510

Here's an example of wording from .NET:
https://github.com/Azure/azure-sdk-for-net/blob/1d49335c35d20bc812107a6e67187cab58b9d58f/sdk/keyvault/Azure.Security.KeyVault.Secrets/src/KeyVaultSecret.cs#L10

From Java:
https://learn.microsoft.com/en-us/java/api/com.azure.security.keyvault.secrets.models.keyvaultsecret?view=azure-java-stable
> Secret is the resource consisting of name, value and its attributes specified in [SecretProperties](https://learn.microsoft.com/en-us/java/api/com.azure.security.keyvault.secrets.models.secretproperties?view=azure-java-stable). It is managed by Secret Service.